src/test/java/teammates/common/datatransfer/questions/FeedbackRubricQuestionDetailsTest.java

```diff
@@ -50,6 +50,60 @@ public void testValidateQuestionDetails_validWeightListSize_errorListShouldBeEmp
         assertEquals(0, errors.size());
     }
 
+    @Test
+    public void testValidateQuestionDetails_invalidDescriptionSize() {
```

Comment:
```suggestion
    public void testValidateQuestionDetails_invalidDescriptionSize_errorReturned() {
```
For consistency in test naming.